src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -259,10 +262,21 @@ const ActionsPane = (props: {contentHeight: string}) => {
   }, [monacoEditor]);
 
   const diffSelectedResource = useCallback(() => {
+    if (!kubeconfigContext || kubeconfigContext === '') {
+      const alert: AlertType = {
+        type: AlertEnum.Error,
+        title: 'Diff failed',
```

Comment:
perhaps 'Diff not available' is a better title

Reply:
resolved